doc/introduction.rst

```diff
@@ -35,6 +35,9 @@ Then play with your Github objects::
         # to see all the available attributes and methods
         print(dir(repo))
 
+To close connections after use::
+
+    g.close()
 Download and install
```

Comment:
```suggestion

Download and install
```

Reply:
Fixed in https://github.com/PyGithub/PyGithub/pull/2724/commits/63956924eaab9ab233bff9b6503c2275d85596cb